sdk/security/keyvault/azkeys/constants.go

```diff
@@ -1,14 +1,10 @@
-//go:build go1.18
-// +build go1.18
-
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
-// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
-// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+// Code generated by Microsoft (R) Go Code Generator. DO NOT EDIT.
 
 package azkeys
 
-// CurveName - Elliptic curve name.
+// CurveName - Elliptic curve name. For valid values, see JsonWebKeyCurveName.
```

Comment:
```suggestion
// CurveName - Elliptic curve name.
```

I guess that slug is getting added from TypeSpec, but aren't all the valid values already constants below? This shows up in a few spots. It doesn't even seem like the TypeSpec should mention it.

Reply:
fixed in d09d2b5